src/website/src/app/documentation/demos/datepicker/demos/datepicker-date-io.demo.html

```diff
@@ -8,11 +8,12 @@
 </clr-alert>
 <form clrForm class="datepicker-demo" clrLayout="vertical">
     <clr-date-container>
+        <label for="dateObject">Date Object</label>
```

Comment:
No `for` should be provided in the demos unless you explicitly define the `id` and want to (like for testing). Should remove them from our docs and samples.

Reply:
removed.

Reply:
Still here?